tests/virtualization/virt_install.pm

```diff
@@ -19,7 +19,7 @@ sub run {
     x11_start_program('xterm');
     become_root;
     script_run('virt-install --name TESTING --memory 512 --disk none --boot cdrom --graphics vnc &', 0);
-    wait_still_screen(3);
+    wait_still_screen(15);
```

Comment:
As I'm not a fan of increasing timeouts with magic numbers, can we come up with something better here? As I don't see verification run, but I believe some WM manager is running, we may always wait for 15 seconds cause of blinking cursor or watches. So maybe we can sync on some process we are waiting to finish, etc.?

Reply:
@rwx788 it is just that "virt-install" take a longer time on aarch64 than on x86_64 to show up the virt-viewer window. And if we start VNC too early, it fails.

Reply:
Ah, I've missed that we have 0 timeout in the script run. We still could have tried to introduce smarter check, but let's leave it for later then.

Reply:
woah, 15s is very long for a stilltime (not timeout). This means that we waste up to 30 seconds doing exactly nothing to ensure the screen has not changed. Why 30s? Because we run it in a graphical session and we have a clock on the screen which may switch over the minute meaning that another try for 15s still screen is triggered. I recommend to revisit this change. How about an assert_screen instead?

Reply:
@okurz, I used wait_still_screen because this is what is in place so far. But an assert_screen would be better I think.